### ISSUE TITLE
Use global reCAPTCHA variant

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#13389]
We use reCAPTCHA v2 in addons-server by calling a script (falling back to iframe) from `https://www.google.com/recaptcha/`.

We've had users complaining that it doesn't work in China, and it sounds like there is [a simple solution for this](https://developers.google.com/recaptcha/docs/faq#can-i-use-recaptcha-globally):
> (...) use "www.recaptcha.net" in your code in circumstances when "www.google.com" is not accessible.

So we should try that. According to https://www.lowendtalk.com/discussion/comment/3058885/#Comment_3058885 we should also alter our CSP to not just replace `https://www.google.com/recaptcha/` with `https://www.recaptcha.net/recaptcha/` but also include `https://www.gstatic.cn/recaptcha/`.

**Comment by diox** [comment]
QA: make sure captcha still works in devhub ; ideally testing from different countries if you have VPN available. We'll rely on community for further testing in other countries.

**Comment by AlexandraMoga** [comment]
I've verified this on -dev by accessing the developer agreement page with a new user, from various IPs (using a VPN). I've managed to cover European countries as well as some from Asia and the Americas but I couldn't verify for China specifically.
Apparently it is not so easy to connect to China, even by using paid VPN services.

For the countries verified, the reCAPTCHA continues to work as expected.
